core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
+                    old_value: null,
+                    question_dict: {
+                      id: '1',
+                      question_state_data: {
+                        content: {
+                          html: 'Question 2',
+                          content_id: 'content_2'
+                        },
+                        interaction: {
+                          answer_groups: [{
+                            outcome: {
+                              dest: 'outcome 1',
+                              dest_if_really_stuck: null,
+                              feedback: {
+                                content_id: 'content_1',
+                                html: ''
+                              },
+                              labelled_as_correct: true,
+                              param_changes: [],
+                              refresher_exploration_id: null
+                            },
+                            rule_specs: [],
+                          }],
+                          confirmed_unclassified_answers: [],
+                          customization_args: {
+                            placeholder: {
+                              value: {
+                                content_id: 'ca_placeholder_0',
+                                unicode_str: ''
+                              }
+                            },
+                            rows: { value: 1 },
+                            catchMisspellings: {
+                              value: false
+                            }
+                          },
+                          default_outcome: {
+                            dest: null,
+                            dest_if_really_stuck: null,
+                            feedback: {
+                              html: 'Correct Answer',
+                              content_id: 'content_2'
+                            },
+                            param_changes: [],
+                            labelled_as_correct: true
+                          },
+                          hints: [{
+                            hint_content: {
+                              html: 'Hint 1',
+                              content_id: 'content_3'
+                            }
+                          }],
+                          solution: {
+                            correct_answer: 'This is the correct answer',
+                            answer_is_exclusive: false,
+                            explanation: {
+                              html: 'Solution explanation',
+                              content_id: 'content_4'
+                            }
+                          },
+                          id: 'TextInput'
+                        },
+                        param_changes: [],
+                        recorded_voiceovers: {
+                          voiceovers_mapping: {}
+                        }
+                      },
+                    },
```

Comment:
A translation suggestion change shouldn't have a `question_dict`. It also shouldn't have a `skill_id`, so we may need to make that property optional in the `SuggestionChangeBackendDict` interface in core/templates/domain/suggestion/suggestion.model.ts. Ditto below.

Reply:
Done